lib/views/location_card.dart

```diff
@@ -12,117 +18,143 @@ class LocationCard extends StatefulWidget {
-                  ),
-                  fit: BoxFit.cover,
+          return Container(
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage(
+                  (snapshot.data!.isDayTime)
+                      ? 'assets/day.jpg'
+                      : 'assets/night.jpg',
                 ),
+                fit: BoxFit.cover,
               ),
-              child: Padding(
-                padding: const EdgeInsets.fromLTRB(0.0, 70.0, 0.0, 0.0),
-                child: Column(
-                  children: [
-                    const SizedBox(height: 20.0),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      crossAxisAlignment: CrossAxisAlignment.center,
-                      children: [
-                        Text(
-                          widget.location.location,
-                          style: const TextStyle(
-                            letterSpacing: 2.0,
-                            color: Colors.white,
-                          ),
-                        ),
-                      ],
-                    ),
-                    const SizedBox(height: 20.0),
-                    widget.location.time != null
-                        ? Text(
-                            widget.location.time ?? '',
-                            style: TextStyle(
-                              fontSize: 66.0,
-                              color: Colors.white,
-                            ),
-                          )
-                        : const SpinKitSpinningLines(
-                            color: Colors.blue,
-                            size: 80.0,
-                          ),
-                    Expanded(
-                      child: Center(
-                        child: ListView(
-                          scrollDirection: Axis.horizontal,
-                          padding: const EdgeInsets.all(8),
-                          children: <Widget>[
-                            Container(
-                              width: 200,
-                              child: Align(
-                                alignment: Alignment.topCenter,
-                                child: Text(
-                                  'Day of the week: ${widget.location.dayOfWeek.toString()}',
-                                  style: TextStyle(
-                                    color: Colors.white,
-                                  ),
+            ),
+            child: Padding(
+              padding: const EdgeInsets.fromLTRB(0.0, 70.0, 0.0, 0.0),
+              child: DefaultTabController(
+                length: tabs.length,
+                child: NestedScrollView(
+                  headerSliverBuilder: (context, innerBoxIsScrolled) {
+                    return <Widget>[
+                      SliverList(
+                        delegate: SliverChildListDelegate([
+                          Column(
```

Comment:
Sometimes, when the indentation grows with more widgets, you can break them up in the build function.

like:
```dart
// Column starting here at line 70
Widget result = Column(...);

result = NestedScrollView(
  ...
  result // Placing the Column in the SliverList
);

return result;
```

It does not have to be exactly like that, but breaking it up into sections of widgets can make it more readable. 

Alternatively, when the indentation starts to grow, you can split sections up into methods, like:

```dart
NestedScrollView(
  headerSliverBuilder: _getHeader,
  body: _getBody,
)
```

Lastly, you could also break sections out into individual widgets.